### DISCUSSION TITLE
docker部署的版本，支持2022-blake3-aes-128-gcm协议

### DISCUSSION THREAD

**Discussion by chad122** [v2rayA/v2rayA#1486] [Q&A]
目前最新版的docker镜像，不支持2022-blake3-aes-128-gcm，可否出个新的版本。

**Comment by chad122** [comment]
我尝试基于最新的v2fly-core打了一个镜像，启动后会报错：检测到 geosite.dat, geoip.dat 文件或 v2ray-core 可能未正确安装，请检查

**Comment by xingshu1990** [comment]
可能配置了V2RAYA_V2RAY_BIN=/usr/local/bin/v2ray  把这句删了或者注释了，重新启用试试。
